packages/manager/modules/pci/src/projects/new/components/support-ticket-banner/template.html

```diff
@@ -13,6 +13,7 @@
     <a
         class="d-block"
         data-ng-href="{{:: $ctrl.supportTicketLink}}"
+        data-ng-click="$ctrl.onContactUsLinkClick()"
```

Comment:
Why not use `track-click` directive directly ?

Reply:
It's another way to complete it.
For me we have more control to do things with a callback function and it's more scalable.